transform/settings.py

```diff
@@ -10,9 +10,9 @@
 SDX_SEQUENCE_URL = os.getenv("SDX_SEQUENCE_URL", "http://sdx-sequence:5000")
 
 FTP_HOST = os.getenv("FTP_PATH", "\\\\NP3RVWAPXX370\\SDX_preprod\\")
```

Comment:
In the readme the environment variable is FTP_HOST but here is FTP_PATH. Does this need to match up?

Reply:
I took a look, and the answer to this is yes and no.  The data out of the config is the important bit and that's correct (ftp_path).  The fact that it's assigned to FTP_HOST and called that within the application is confusing so yes it should change.  I'll make a card to fix this because I'd rather wait until the removal of sdx-common happens because that DOES look at FTP_HOST (which is really ftp_path...) so changing it now might accidentally break something.